graaljs/graaljs-gradle-webpack-guide/README.md

```diff
@@ -0,0 +1,187 @@
+# Using Node Packages in a Java Application with Gradle
+
+## Introduction
+This guide demonstrates how to integrate JavaScript libraries into a Java application using **GraalJS** and the **GraalVM Polyglot API**. Specifically, we will use the `qrcode` NPM package to generate QR codes within a Java application, leveraging GraalVM's ability to execute JavaScript code.
+
```

Comment:
I think it would be nice to link the Maven and Gradle guides with each other. Could you please add cross-references and tell users that there is a Gradle version of the guide in the Maven guide and vice-versa?